plugins/catalog-backend-module-bitbucket/src/lib/BitbucketRepositoryParser.ts

```diff
@@ -15,13 +15,12 @@
  */
 
 import { BitbucketIntegration } from '@backstage/integration';
+import {
+  CatalogProcessorResult,
+  processingResult,
+} from '@backstage/plugin-catalog-backend';
 import { Logger } from 'winston';
-import { CatalogProcessorResult, processingResult } from '../../../api';
 
-/**
- * @public
- * @deprecated type inlined.
- */
 export type BitbucketRepositoryParser = (options: {
```

Comment:
At the changeset you wrote that the type was removed but it still exists here (not "public" anymore though). Is this intended?

Reply:
Yeah good point, it no longer is in use. But you know what, now that we've this in a separate module, I think we should instead be reintroducing the type since it makes perfect sense in this context and the noise level is so low. I'll actually add it in instead!